src/frontend/web_application/src/components/MenuBar/index.jsx

```diff
@@ -1,26 +1,61 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
+import NavList, { ItemContent } from '../../components/NavList';
+import Link from '../../components/Link';
+
 
 import './style.scss';
 
-const MenuBar = ({ className, children }) => {
-  const menuBarClassName = classnames(
-    'm-menu-bar',
-    className,
-  );
 
-  return <div className={menuBarClassName}>{children}</div>;
-};
+class MenuBar extends Component {
+  static propTypes = {
+    className: PropTypes.string,
+    children: PropTypes.node,
+    navLinks: PropTypes.arrayOf(PropTypes.shape({})),
+  };
+
+  static defaultProps = {
+    className: null,
+    children: null,
+    navLinks: null,
+  };
+
+  state = {
+    activeLink: {},
+  };
+
+  render() {
+    const { className, navLinks, children } = this.props;
+    const menuBarClassName = classnames(
+      'm-menu-bar',
+      className,
+    );
 
-MenuBar.propTypes = {
-  className: PropTypes.string,
-  children: PropTypes.node,
-};
+    return (
+      <div className={menuBarClassName}>
+        {navLinks &&
+          <NavList>
+            {navLinks.map(link => (
+              <ItemContent
+                active={this.state.activeLink === link && true}
```

Comment:
I suppose this doesn't work.

Actually there is an other way to do that but it is a bit more complex: it can be done by checking if link's path match the current pathname in redux store. So this component become "connected" and I'm not yet sure if should leave this as dump component or not.

may be you can add a FIXME comment until it is not working. (we should add an issue "find & fix FIXME in github ;) )